### PR TITLE
Feature flagging of add/edit phrases

### LINKED ISSUE #269: Refactor: Feature Flag Add Phrases
Creates a new environment variable for `AddPhraseEnabled`
Minor cleanup of `EditCategoriesDetailViewController`
Presents the EditTextViewController for adding a phrase.

### PR DESCRIPTION
**Pull request by stevefosterwta**
- There is a single path to adding phrases to custom categories that is now behind the EditPhrasesEnabled environment variable. The path is `Settings` -> `Categories` -> `Your Custom Category` -> `MISSING_LOCALIZTION_Add Phrases.`
- Refactored `EditPhrases` to handle editing any category. It was previously hardcoded to just 'My Sayings.'

**Notes for Test Engineering:**
- It is possible to successfully add phrases to a custom category those phrases are not showing up when selecting the custom category in the main screen. Updating that screen is part of the refactor and out of scope for this PR.
- The refactoring of `EditPhrases` should be tested in My Sayings and following the user path outlined above.

**Xcode Environment Variables:** 
image

### CODE REVIEW

File: Vocable/Features/Settings/EditCategories/EditCategoryDetailViewController.swift [hunk 1]
```diff
@@ -153,41 +136,69 @@ class EditCategoriesDetailViewController: UIViewController, UICollectionViewDele
...
+
         let shouldShowCategory = !cell.showCategorySwitch.isOn
         category.setValue(!category.isHidden, forKey: "isHidden")
         cell.showCategorySwitch.isOn = shouldShowCategory
         saveContext()
     }
-    
+
+    private func displayEditPhrasesViewController() {
+        if let viewController = UIStoryboard(name: "EditPhrases", bundle: nil).instantiateViewController(identifier: "MyPhrases") as? EditPhrasesViewController {
```
Comment (Clstroud) [inline Vocable/Features/Settings/EditCategories/EditCategoryDetailViewController.swift]:
```suggestion
        let viewController = UIStoryboard(name: "EditPhrases", bundle: nil).instantiateViewController(identifier: "MyPhrases") as! EditPhrasesViewController
```

File: Vocable/Features/Settings/EditCategories/EditCategoryDetailViewController.swift [hunk 2]
```diff
@@ -153,41 +136,69 @@ class EditCategoriesDetailViewController: UIViewController, UICollectionViewDele
...
+            viewController.category = category
+            show(viewController, sender: nil)
+        }
```
Comment (Clstroud) [inline Vocable/Features/Settings/EditCategories/EditCategoryDetailViewController.swift]:
```suggestion
        viewController.category = category
        show(viewController, sender: nil)
        
```

File: Vocable/Features/Settings/SelectionMode/SelectionModeCollectionViewController.swift [hunk 3]
```diff
@@ -11,11 +11,12 @@ import UIKit
 final class SelectionModeCollectionViewController: UICollectionViewController {
     
     private enum SelectionModeItem: String, Hashable {
+
+        case headTrackingToggle = "Head Tracking"
```
Comment (Clstroud) [inline Vocable/Features/Settings/SelectionMode/SelectionModeCollectionViewController.swift]: Note: Review localization on this

**Review by Clstroud — APPROVED** [review]
Looks good, just a few minor suggestions/notes

**Comment by stechkev** [comment]
"Add new category" button should be controlled by the same feature flag

(now https://github.com/willowtreeapps/vocable-ios/issues/283)

**Pull request merged**